client-cmds/qlean-cmd.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
 #-----------------------qlean setup----------------------
-node_binary=
+# expects "qlean" submodule or symlink inside "lean-quickstart" root directory
+# https://github.com/qdrvm/qlean-mini
+node_binary="$scriptDir/qlean/build/src/executable/qlean \
+      --modules-dir $scriptDir/qlean/build/src/modules \
+      --genesis $configDir/config.yaml \
+      --validator-registry-path $configDir/validators.yaml \
+      --bootnodes $configDir/nodes.yaml
+      --node-id $item --node-key /config/$privKeyPath \
+      --listen-addr /ip4/0.0.0.0/udp/$quicPort/quic-v1 \
+      2>&1 | tee $dataDir/$item.log"
```

Comment:
I assumed that capturing logs to file helps debugging and should be enabled for all nodes.
Please don't add this to `node_binary`.
```suggestion
```

Reply:
nodes should auto gen logs in data dir for example zeam writes file logs with debug level while console logs are only info level

so its upto the client what they want and hence in a particular client cmd